Dockerfile

```diff
@@ -14,14 +9,14 @@ RUN go mod download
 
 COPY . .
 ARG version
-ENV version=${version}
-RUN ./scripts/build-engine.sh
+ENV PATCH_VERSION=${version}
```

Comment:
Why put this env `PATCH_VERSION`? it is not used (except in makefile, is it why?)